### PR TITLE
[skip ci] PEP8 amendments on polyhedron.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Whitespace amendments again. Thanks. :)

### CODE REVIEW

File: sympy/combinatorics/polyhedron.py [hunk 1]
```diff
@@ -336,11 +336,10 @@ def __new__(cls, corners, faces=[], pgroup=[]):
...
             12 faces:
 
-            (0,1,2,3,4)
-            (0,1,6,10,5) (1,2,7,11,6) (2,3,8,12,7) (3,4,9,13,8) (0,4,9,14,5)
-            (5,10,16,15,14) (
-                6,10,16,17,11) (7,11,17,18,12) (8,12,18,19,13) (9,13,19,15,14)
-            (15,16,17,18,19)
+            (0, 1, 2, 3, 4)
+            (0, 1, 6, 10, 5) (1, 2, 7, 11, 6) (2, 3, 8, 12, 7) (3, 4, 9, 13, 8) (0, 4, 9, 14, 5)
+            (5, 10, 16, 15, 14) (6, 10, 16, 17, 11) (7, 11, 17, 18, 12) (8, 12, 18, 19, 13) (9, 13, 19, 15, 14)
```
Comment (jcrist) [inline sympy/combinatorics/polyhedron.py]: Please wrap to 80 columns or less.

Comment (jksuom) [inline sympy/combinatorics/polyhedron.py]: This might be one example where adding whitespace would not improve the readability.

**Comment by debugger22** [comment]
You should also squash commits.

**Comment by moorepants** [comment]
I don't think squashing is necessary. Merging.

**Pull request merged**